.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+max-line-length = 120
+per-file-ignores =
+     pyiddidf/test/test_idd_processor.py: W291
```

Comment:
Configure flake8, and we add an exception here for trailing spaces in the idd processing unit test code because we are testing trailing spaces in string blocks there.